tools/dependencies-report/src/main/resources/notices/ffi-binary-libfixposix-NOTICE.txt

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
this gem is [tri-licensed (`Ruby`, `EPL-2.0`, `LGPL-2.1-or-later`)](https://github.com/byteit101/subspawn/blob/master/subspawn-posix/subspawn-posix.gemspec#L40), and the Ruby license is preferable to EPL so I will swap it out.